docs/source/analysis_objects.rst

```diff
@@ -27,9 +27,9 @@ your app can always be verified. These hashes exist on the following attributes:
 -   ``configuration_values_hash``
 -   ``configuration_manifest_hash``
 
-If an input or configuration attribute is ``None``, so will its hash attribute be. For ``Manifest``s, some metadata
-about the ``Datafile``s and ``Dataset``s within them, and about the ``Manifest`` itself, is included when calculating
-the hash:
+If an input or configuration attribute is ``None``, so will its hash attribute be. For ``Manifest``, some metadata
+about the ``Datafile`` and ``Dataset`` instances within them, and about the ``Manifest`` itself, is included when
+calculating the hash:
```

Comment:
I only changed it because it seems to be invalid RST to put a letter directly after preformatted text (i.e. "s" in this case). I try to keep the "s" outside of the preformatting because the class names aren't plural. Should I put a space between the preformatted text and the "s" or do something else?

Reply:
No, if it's a rst annoyance then don't worry about it. Man, I hate RST, but it's just so much more powerful than MD. :(

Reply:
I could just give up a little purity and put the "s" inside the preformatted text?